db/migrate/20200604145556_add_listing_to_orders.rb

```diff
@@ -0,0 +1,5 @@
+class AddListingToOrders < ActiveRecord::Migration[6.0]
+  def change
+    add_reference :orders, :listing, null: false, foreign_key: true
```

Comment:
same thing here: not every order will have a listing
so null: false should be removed? 
read up on this: https://stackoverflow.com/questions/32305821/rails4-migration-adding-null-false-to-an-add-reference-method
maybe just need to remove `null: false`

Reply:
sorry my bad! all orders will hhave a listing haha